docs/index.rst

```diff
@@ -46,6 +46,50 @@ Note that by changing the first two lines to::
 you could use the code exactly the same way, this time storing data inside a
 Redis database.
 
+Store factories (from configurations)
+=====================================
+
+There are two possibilities to generate stores from configurations:
+
+1) Use a dictionary with configuration data (e.g. loaded from an ini file)
+
+.. code-block:: python
+
+    from minimalkv import get_store
+
+    params = {
+        'account_name': 'test',
+        'account_key': 'XXXsome_azure_account_keyXXX',
+        'container': 'my-azure-container',
+    }
+    store = get_store('azure', **params)
+    store.put(u'key', b'value')
+    assert store.get(u'key') == b'value'
+
+2) Use an URL to specify the configuration
+
+.. code-block:: python
+
+    from minimalkv import get_store_from_url, get_store
+
+    store = get_store_from_url('azure://test:XXXsome_azure_account_keyXXX@my-azure-container')
+    store.put(u'key', b'value')
+    assert store.get(u'key') == b'value'
+
+URL and store types:
+
+* In memory: :code:`memory://` and :code:`hmemory://`.
```

Comment:
Maybe add examples here?

Reply:
These are the explicit examples, the full URL is just `memory://` with no further paths.